src/gulpfile.js

```diff
@@ -160,7 +160,10 @@ gulp.task('common:build', ['clean'], function() {
     return gulp.start('copy:vendor-css', 'copy:vendor-images', 'sass', 'collectstatic');
 });
 
-gulp.task('test', sequence(['jshint',
+gulp.task('test', sequence([
+            'development',
+            'production',
```

Comment:
Does having both development and production together make sense? They both run clean so production is just deleting everything that development produced.